test/integration/consumer_group_test.exs

```diff
@@ -24,6 +24,26 @@ defmodule KafkaEx.ConsumerGroup.Test do
     assert consumer_group == :no_consumer_group
   end
 
+  test "create_worker allows us to pass in use_ssl and ssl_options options" do
+    Application.put_env(:kafka_ex, :use_ssl, true)
+    ssl_options = Application.get_env(:kafka_ex, :ssl_options)
+    assert ssl_options == Config.ssl_options()
+
+    custom_ssl_options = [
+      cacertfile: File.cwd!() <> "/ssl/ca-cert-custom",
+      certfile: File.cwd!() <> "/ssl/cert-custom.pem",
+      keyfile: File.cwd!() <> "/ssl/key-custom.pem"
```

Comment:
Can we just reuse the files we already have?  I'm also not sure how the test would pass if we don't configure the test broker to use these.

Reply:
Yah, these are the same as the existing files we already have, just the filename is different so it can be validated in the test that a different file is being used.

Reply:
Ahh that makes sense.  It would be nice if we could document that somehow (even if it's just in a comment in this test).  There's about a 100% chance that some day someone will wonder why we have a duplicate set of keys with different names and get very confused.  Probably me because I'll forget :joy:.  Another option might be to create a copy of the files as part of the test run and then delete them during clean up.

Reply:
Added some documentation, and realized it would be even more explicit to use a symbolic file links to prevent duplicate files combined with a comment. Would that work?

Reply:
On second though, I also really like your proposal @dantswain of creating and deleting the files to not confuse the repo since others may look at those certs for examples of how to configure client.